bundles/binding/org.openhab.binding.cardio2e/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,29 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.cardio2e.internal
+Ignore-Package: org.openhab.binding.cardio2e.internal
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Name: openHAB Cardio2e Binding
+Bundle-SymbolicName: org.openhab.binding.cardio2e
+Bundle-Vendor: openHAB.org
+Bundle-Version: 1.7.0.qualifier
+Bundle-ManifestVersion: 2
+Bundle-Description: This is the Cardio2e binding of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: org.apache.commons.lang,
+ org.openhab.core.binding,
+ org.openhab.core.events,
+ org.openhab.core.items,
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.types,
+ org.openhab.model.item.binding,
+ org.osgi.framework,
+ org.osgi.service.cm;version="1.4.0",
+ org.osgi.service.component,
+ org.osgi.service.event,
+ org.slf4j
+Export-Package: org.openhab.binding.cardio2e
+Bundle-DocURL: http://www.openhab.org
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
```

Comment:
Cannot export plugin if change to "Bundle-RequiredExecutionEnvironment: JavaSE-1.7", so I have not changed it for now. I will do it later, when I find the problem of exporting the plugin.
I also installed lastest available Java JDK (1.8.121) and changed Eclipse Workspace compiler compliance level to match Java 8. My openHAB IDE is based on version 1.7.0.
When I tried to export the plugin, I received the following error:
Processing inclusion from feature org.eclipse.pde.container.feature: Bundle org.openhab.binding.cardio2e_1.10.0.qualifier failed to resolve.:
	Missing Constraint: Bundle-RequiredExecutionEnvironment: JavaSE-1.8
	Host plug-in JavaSE_0.0.0 has not been found.